src/game/savegame/savegame.c

```diff
@@ -419,8 +420,6 @@ bool Savegame_Save(int32_t slot_num, GAME_INFO *game_info)
         sprintf(
             &req->item_texts[req->item_text_len * slot_num], "%s %d",
             g_GameFlow.levels[g_CurrentLevel].level_title, g_SaveCounter);
-        g_SavedGamesCount++;
-        g_SaveCounter++;
```

Comment:
Why are these removed, won't they cause the sg counter to stay stuck?

Reply:
Sorry I forgot to leave a comment on this. Basically these increments are worthless because these counters are set to 0 and recalculated in the scan savegame function which is called at the end of the sage function. All they did was cause me confusion lol.